### PR TITLE
add translations

### PR DESCRIPTION
**Pull request by dvm1984**
#### What is the purpose of this pull request?
added translations

### CODE REVIEW

**Review by alvimm — DISMISSED** [review]

File: messages/ru.json [hunk 1]
```diff
@@ -4,30 +4,30 @@
...
-  "address-form.geolocation.example.BRA": "Например: Av Paulista, 1578, São Paulo",
-  "address-form.geolocation.example.CAN": "Например: 450 Wilbrod St, Ottawa",
-  "address-form.geolocation.example.CHL": "Например: Los Militares, 6191, Santiago",
-  "address-form.geolocation.example.COL": "Например: Calle 93 # 14-20, Bogotá",
-  "address-form.geolocation.example.ECU": "Например: Av Rio Amazonas, N 37-61, Quito",
-  "address-form.geolocation.example.ESP": "Например: Calle Fernando el Santo, 6, Madrid",
-  "address-form.geolocation.example.GTM": "Например: 6A Calle 6, Guatemala",
-  "address-form.geolocation.example.MEX": "Например: Calle de Tacuba 8, Ciudad de México",
+  "address-form.geolocation.example.ARG": "Eg: Av. del Libertador 1473, Buenos Aires",
+  "address-form.geolocation.example.BOL": "Eg: Например: Av Arce, 2556, La Paz",
```
Comment (lucasecdb) [inline messages/ru.json:8]: shouldn't this "Например" be removed?
```suggestion
  "address-form.geolocation.example.BOL": "Eg: Av Arce, 2556, La Paz",
```

File: messages/ru.json [hunk 2]
```diff
@@ -4,30 +4,30 @@
...
+  "address-form.geolocation.example.BRA": "Eg: Av Paulista, 1578, São Paulo",
+  "address-form.geolocation.example.CAN": "Eg: 450 Wilbrod St, Ottawa",
+  "address-form.geolocation.example.CHL": "Eg: Los Militares, 6191, Santiago",
+  "address-form.geolocation.example.COL": "Eg: Calle 93 # 14-20, Bogotá",
+  "address-form.geolocation.example.ECU": "Eg: Av Rio Amazonas, N 37-61, Quito",
+  "address-form.geolocation.example.ESP": "Eg: Calle Fernando el Santo, 6, Madrid",
+  "address-form.geolocation.example.GTM": "Eg: 6A Calle 6, Guatemala",
+  "address-form.geolocation.example.MEX": "Eg: Calle de Tacuba 8, Ciudad de México",
   "address-form.geolocation.example.PER": "Например: Av. José Pardo, 850, Miraflores, Lima",
```
Comment (lucasecdb) [inline messages/ru.json:17]: this also should be updated, right?
```suggestion
  "address-form.geolocation.example.PER": "Eg: Av. José Pardo, 850, Miraflores, Lima",
```

File: messages/ru.json [hunk 3]
```diff
@@ -4,30 +4,30 @@
...
+  "address-form.geolocation.example.USA": "Eg: 225 East 41st Street, New York",
+  "address-form.geolocation.example.VEN": "Eg: Avenida Mohedano, Caracas",
+  "address-form.geolocation.example.ROU": "Eg: Bulevardul Ion Mihalache, București 011192, Romania",
+  "address-form.geolocation.example.BGR": "Eg: улица Елин Пелин 13, София",
+  "address-form.geolocation.example.DEU": "Eg: Rheinalleee 95, Düsseldorf",
+  "address-form.geolocation.example.FRA": "Eg: 7 Rue Hautefeuille, Paris",
+  "address-form.geolocation.example.GBR": "Eg: 29 High Street Poole BH15 1AB",
+  "address-form.geolocation.example.ITA": "Eg: Corso Vittorio Emanuele 69, Milano",
+  "address-form.geolocation.example.NLD": "Eg: Surinamestraat 27, Amsterdam",
   "address-form.geolocation.example.POL": "Например: Ulica Twarda 3, Szczecin",
```
Comment (lucasecdb) [inline messages/ru.json:31]: same as above
```suggestion
  "address-form.geolocation.example.POL": "Eg: Ulica Twarda 3, Szczecin",
```

File: messages/sk.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  "address-form.geolocation.example.PRY": "Eg: Avenida Eusebio Ayala, 100, Assunção",
+  "address-form.geolocation.example.UNI": "Napr. 225 East 41st Street, New York",
+  "address-form.geolocation.example.URY": "Eg: Bulevar Artigas, 1394, Montevidéu",
+  "address-form.geolocation.example.USA": "Napr. 225 East 41st Street, New York",
+  "address-form.geolocation.example.VEN": "Eg: Avenida Mohedano, Caracas",
+  "address-form.geolocation.example.ROU": "Eg: Bulevardul Ion Mihalache, București 011192, Romania",
+  "address-form.geolocation.example.BGR": "Eg: улица Елин Пелин 13, София",
+  "address-form.geolocation.example.DEU": "Eg: Rheinalleee 95, Düsseldorf",
+  "address-form.geolocation.example.FRA": "Eg: 7 Rue Hautefeuille, Paris",
+  "address-form.geolocation.example.GBR": "Napr. 29 High Street Poole BH15 1AB",
```
Comment (lucasecdb) [inline messages/sk.json:28]: these "Napr." vs "Eg:" seem inconsistent in this file

File: messages/da.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  "address-form.geolocation.example.BOL": "F.eks.: Av Arce, 2556, La Paz",
+  "address-form.geolocation.example.BRA": "F.eks.: Av Paulista, 1578, São Paulo",
+  "address-form.geolocation.example.CAN": "F.eks.: 450 Wilbrod St, Ottawa",
+  "address-form.geolocation.example.CHL": "F.eks.: Los Militares, 6191, Santiago",
+  "address-form.geolocation.example.COL": "F.eks.: Calle 93 # 14-20, Bogotá",
+  "address-form.geolocation.example.ECU": "F.eks.: Av Rio Amazonas, N 37-61, Quito",
+  "address-form.geolocation.example.ESP": "F.eks.: Calle Fernando el Santo, 6, Madrid",
+  "address-form.geolocation.example.GTM": "F.eks.: 6A Calle 6, Guatemala",
+  "address-form.geolocation.example.MEX": "F.eks.: Calle de Tacuba 8, Ciudad de México",
+  "address-form.geolocation.example.PER": "F. eks.: Av. José Pardo, 850, Miraflores, Lima",
```
Comment (lucasecdb) [inline messages/da.json:17]:
```suggestion
  "address-form.geolocation.example.PER": "F.eks.: Av. José Pardo, 850, Miraflores, Lima",
```

File: messages/de.json [hunk 6]
```diff
@@ -1,35 +1,35 @@
...
+  "address-form.geolocation.example.CHL": "z. B.: Los Militares, 6191, Santiago",
+  "address-form.geolocation.example.COL": "z. B.: Calle 93 # 14-20, Bogotá",
+  "address-form.geolocation.example.ECU": "z. B.: Av Rio Amazonas, N 37-61, Quito",
+  "address-form.geolocation.example.ESP": "z. B.: Calle Fernando el Santo, 6, Madrid",
+  "address-form.geolocation.example.GTM": "z. B.: 6A Calle 6, Guatemala",
+  "address-form.geolocation.example.MEX": "z. B.: Calle de Tacuba 8, Ciudad de México",
+  "address-form.geolocation.example.PER": "z. B.: Av. José Pardo, 850, Miraflores, Lima",
+  "address-form.geolocation.example.PRT": "z. B.: Av. da França 20, Porto",
+  "address-form.geolocation.example.PRY": "z. B.: Avenida Eusebio Ayala, 100, Assunção",
+  "address-form.geolocation.example.UNI": "z. B.:225 East 41st Street, New York",
```
Comment (lucasecdb) [inline messages/de.json:20]:
```suggestion
  "address-form.geolocation.example.UNI": "z. B.: 225 East 41st Street, New York",
```

File: messages/fr.json [hunk 7]
```diff
@@ -1,68 +1,68 @@
...
+  "address-form.geolocation.example.USA": "Ex : 225 East 41st Street, New York",
+  "address-form.geolocation.example.VEN": "Ex : Avenida Mohedano, Caracas",
+  "address-form.geolocation.example.ROU": "Ex : Bulevardul Ion Mihalache, București 011192, Romania",
+  "address-form.geolocation.example.BGR": "Ex : улица Елин Пелин 13, София",
+  "address-form.geolocation.example.DEU": "Ex : Rheinalleee 95, Düsseldorf",
+  "address-form.geolocation.example.FRA": "Ex : 7 Rue Hautefeuille, Paris",
+  "address-form.geolocation.example.GBR": "Ex : 29 High Street Poole BH15 1AB",
+  "address-form.geolocation.example.ITA": "Ex : Corso Vittorio Emanuele 69, Milano",
+  "address-form.geolocation.example.NLD": "Ex : Surinamestraat 27, Amsterdam",
+  "address-form.geolocation.example.POL": "Ex : Ulica Twarda 3, Szczecin",
```
Comment (lucasecdb) [inline messages/fr.json:31]: I think these whitespaces after "Ex" shouldn't be added to keep it consistent with the other files

**Review by alvimm — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]
✨

**Pull request merged**